activity/activity-ktx/build.gradle

```diff
@@ -30,16 +30,16 @@ dependencies {
     api("androidx.core:core-ktx:1.1.0") {
         because 'Mirror activity dependency graph for -ktx artifacts'
     }
-    api(project(":lifecycle:lifecycle-runtime-ktx")) {
+    api(projectOrArtifact(":lifecycle:lifecycle-runtime-ktx")) {
         because 'Mirror activity dependency graph for -ktx artifacts'
     }
-    api(project(":lifecycle:lifecycle-viewmodel-ktx"))
-    api(project(":savedstate:savedstate-ktx")) {
+    api(projectOrArtifact(":lifecycle:lifecycle-viewmodel-ktx"))
+    api(projectOrArtifact(":savedstate:savedstate-ktx")) {
```

Comment:
i'm afraid this will bite us in presubmit but maybe the solution is to have something similar to affected module detector in github, not sure right now. We plan to move them to here anyways so maybe it will be OK meanwhile.